src/runtime/threads/threadmanager.cpp

```diff
@@ -582,17 +597,24 @@ namespace hpx { namespace threads {
             case resource::shared_priority:
             {
 #if defined(HPX_HAVE_SHARED_PRIORITY_SCHEDULER)
+                std::string affinity_desc;
+                std::size_t numa_sensitive =
+                    hpx::util::get_affinity_description(cfg_, affinity_desc);
                 // instantiate the scheduler
                 typedef hpx::threads::policies::
                     shared_priority_queue_scheduler<>
                         local_sched_type;
                 local_sched_type::init_parameter_type init(
-                    thread_pool_init.num_threads_, {4, 4, 64},
+                    thread_pool_init.num_threads_, {1, 1, 64},
```

Comment:
Did you mean to change these?

Reply:
yes. It was intentional, but I can remove it to a later PR as it is an unrelated flyby.